railties/lib/rails/command.rb

```diff
@@ -106,6 +107,12 @@ def command_type # :doc:
           @command_type ||= "command"
         end
 
+        def modify_args_for_rake(args, namespace)
+          args.flat_map do |arg|
+            HELP_MAPPINGS.include?(arg) ? ["--describe", namespace] : arg
+          end
```

Comment:
Could this be simplified as?

```suggestion
          (args & HELP_MAPPINGS).any? ? ["--describe", namespace] : args
```

Or even?

```suggestion
          HELP_MAPPINGS.include?(args[0]) ? ["--describe", namespace] : args
```

Also, I feel like `modify_args_for_rake` is a bit nondescript.  Would it be any better to inline the method using a post-conditional?  e.g.:

```ruby
args = ["--describe", full_namespace] if HELP_MAPPINGS.include?(args[0])
```

Reply:
I think inlining is more clear and made the change. Thanks!

Reply:
I've also move the test to `railties/test/application/rake_test.rb`